src/pages/Blog/BlogDetails/index.tsx

```
@@ -1,193 +1,109 @@
-
-            <section>
-              <h3 className='text-2xl font-semibold mt-3'>Conclusion</h3>
-              <p className='mt-2 text-gray-700'>
-                Maintaining a nutritious diet during pregnancy is one of the most important steps you can take for
-                yourself and your baby. By incorporating these 10 superfoods into your meals, you’ll not only meet your
-                body’s changing nutritional needs but also support the healthy development of your little one. Remember,
-                every small effort you make today lays the foundation for a brighter, healthier future. Start with small
-                changes, and enjoy this beautiful journey of motherhood!
-              </p>
+            <section className='prose prose-lg max-w-none'>
+              <div dangerouslySetInnerHTML={{ __html: blogData.content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)